lib/krikri/enrichments/genre_filter.rb

```diff
@@ -0,0 +1,46 @@
+module Krikri::Enrichments
+  ##
+  # Enrichment to remove non-genre fields from
+  #
+  #   StripHtml.new.enrich_value('Book') => 'Book'
+  #   StripHtml.new.enrich_value('book') => 'book'
+  #   StripHtml.new.enrich_value('not a book') => nil
+  #
+  # Allowed genre terms are:
+  #
+  #  - Book
+  #  - Film/Video
+  #  - Manuscript
+  #  - Maps
+  #  - Music
+  #  - Musical Score
+  #  - Newspapers
+  #  - Nonmusic
+  #  - Photograph/Pictorial Works
+  #  - Serial
+  #
+  # Removes all non-string values
+  class GenreFilter
```

Comment:
I think we should actually move the whole thing to `Heidrun`.  It's clearly not a core krikri enrichment.